layouts/partials/head/scripts.html

```diff
@@ -9,14 +9,11 @@
 {{ end }}
 
 {{ if $.Param "enableSearch" }}
-  {{ $fuse := resources.Get "js/fuse.min.js" | resources.Fingerprint }}
-  <script defer src="{{ $fuse.RelPermalink }}"></script>
+  <script defer src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.0"></script>
 {{ end }}
 
-{{ $enquire := resources.Get "js/enquire.min.js" | resources.Fingerprint }}
-<script src="{{ $enquire.RelPermalink }}"></script>
-{{ $lazysizes := resources.Get "js/lazysizes.min.js" | resources.Fingerprint }}
-<script defer src="{{ $lazysizes.RelPermalink }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/enquire.js@2.1.6/dist/enquire.min.js"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/lazysizes@5.2.2/lazysizes.min.js"></script>
```

Comment:
It makes it easier to update. Btw, it would be good to use the https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity here, to promote good practices.

(I'm not a maintainer of this plugin, but I plan to reuse your MR for my blog)

Reply:
You can set variables for the URLs in some sort of variable and update them. How is it easier to update local files?

Reply:
> It makes it easier to update.

Your change makes it easier to update in the future (just bump the version without changing the - outdated - files in the local repo).

The suggestion with the integrity check is independent.